packages/flutter_tools/lib/src/base/args.dart

```diff
@@ -0,0 +1,35 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:args/args.dart';
+
+/// A typed wrapper for the flutter command [ArgResults].
+class TypedArgResults {
+  TypedArgResults(this._argResults);
+
+  final ArgResults _argResults;
+
+  /// Returns the value of the command line flag `name`.
+  bool getFlag(String name) => _argResults[name] as bool;
```

Comment:
drive by comment - consider naming these `flag`, `option`, and `multiOption`, or maybe with the prefix `read`?

https://www.dartlang.org/guides/language/effective-dart/design#avoid-starting-a-method-name-with-get

Reply:
Updated these to readFlag, readOption, readMultiOption. I can see just flag, option, and multiOption reading well - though I'm never sure about methods without verbs.